e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/BSS-EWCS/MIN-future-cancellation-updates-portal.spec.js

```diff
@@ -13,16 +13,24 @@ describe('Deal Cancellation status updates', () => {
   const dealFacilities = [];
 
   before(() => {
-    cy.insertOneDeal(generateMinReadyToSubmit(), BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-      dealId = deal._id;
-
-      const { mockFacilities } = deal;
-
-      cy.createFacilities(dealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
-        dealFacilities.push(...createdFacilities);
+    cy.insertOneDeal(generateMinReadyToSubmit(), BANK1_MAKER1)
+      .then((insertedDeal) => {
+        deal = insertedDeal;
+        dealId = deal._id;
+
+        const { mockFacilities } = deal;
+
+        cy.createFacilities(dealId, mockFacilities, BANK1_MAKER1)
+          .then((createdFacilities) => {
+            dealFacilities.push(...createdFacilities);
+          })
+          .then(() => {
+            cy.wrap(dealFacilities).should('not.be.empty');
+          });
+      })
+      .then(() => {
+        cy.wrap(deal).should('not.be.empty');
```

Comment:
Do we need to make this change, can we revert? This will make https://github.com/UK-Export-Finance/dtfs2/pull/3594 harder to merge which completely removes the need to use `cy.insertOneDeal`

Reply:
Just testing out